views/includes/page-templates/relevant-period.html

```diff
@@ -0,0 +1,6 @@
+{% set start_date = "28 February 2022" %}
+{% set registration_date = dateMacros.formatDate(dateOfCreation["day"], dateOfCreation["month"], dateOfCreation["year"]) %}
+
+<p class="govuk-body-m">
```

Comment:
I don't think `class="govuk-body-m"` is doing anything so safe to remove

Reply:
that class is doing margin on bottom 20px just like is in describe in prototype. Please check prototype and you will see they are using this class.